backend/app/adapter/facebook/api.go

```diff
@@ -0,0 +1,42 @@
+package facebook
+
+import (
+	"short/app/entity"
+
+	fb "github.com/huandu/facebook"
+)
+
+// API access user's profile information through FB Graph API.
```

Comment:
```suggestion
// API accesses user's profile information through FB Graph API.
```

Reply:
I wonder is this Graph or GraphQL?

Reply:
I know it's super confusing but facebook calls their public API Graph API and it has nothing to do with GraphQL 😕